### PR TITLE
show contributor count on stats page

### PR DESCRIPTION
**Pull request by robertylewis**

### CODE REVIEW

File: templates/mathlib_stats.html [hunk 1]
```diff
@@ -5,18 +5,18 @@
...
 
     <table class="table table-borderless table-sm mt-4 ml-5" style="font-size: 1.25rem">
         <tr>
             <th>Definitions</th>
             <th>Theorems</th>
-            <th>Meta declarations</th>
+            <th>Contributors</th>
         </tr>
         <tr>
             <td>{{ num_defns }}</td>
```
Comment (eric-wieser) [inline templates/mathlib_stats.html:17]: Possibly
```suggestion
            <td>{{ num_defns }} (+ {{num_meta}} meta)</td>
```
Could link `meta` to https://leanprover-community.github.io/extras/tactic_writing.html, although it's not a perfect match.

Comment (robertylewis) [reply templates/mathlib_stats.html:17]: IMO this is more clutter than it's worth, not sure the meta number is particularly interesting to most people.

Comment (bryangingechen) [reply templates/mathlib_stats.html:17]: We could hide more detailed statistics inside a `<details>` dropdown.

Comment (bryangingechen) [reply templates/mathlib_stats.html:17]: Probably something for another PR though.

**Pull request merged**